### DISCUSSION TITLE
How do I set the "Inverted Inputs" attribute of D-Flip-flop in generic circuit ?

### DISCUSSION THREAD

**Discussion by lhcwha** [hneemann/Digital#622] [Q&A] [answered]
How do I set the "Inverted Inputs" attribute of D-Flip-flop in generic circuit ?
I write the following code in the "generic parameterization" field of D-Flip-flop, but it cannot work and error message appears!  
  this.inverterConfig += "[Clr,D]";

What is the syntax of the generic parameterization code?  Is there any documentation for this. I cannot find it.
Please help! Thank you.

**Comment by hneemann** [comment, marked as answer]
I have overseen that! This is impossible right now. You will always get a type error!

**Reply by lhcwha** [reply]
Thank you for your answer.

**Reply by hneemann** [reply]
In the latest [pre-release](https://infdigital.dhbw-mosbach.de/) you can set the inverter config like:

```
this.inverterConfig=newList("D");
```

**Reply by lhcwha** [reply]
Ok. Thank you very much!
